packages/website/translations/english.json

```diff
@@ -92,6 +92,9 @@
     "ORDER_BASICS_DESCRIPTION": "Tutorial on how to create, validate and fill an order using 0x",
     "USE_NETWORKED_LIQUIDITY": "use networked liquidity",
     "USE_NETWORKED_LIQUIDITY_DESCRIPTION": "Learn how to tap into networked liquidity using the Standard Relayer API",
+    "INTEGRATE_0X_INSTANT": "integrate 0x Instant and AssetBuyer",
```

Comment:
The other titles feel like a description of what you're going to do, as opposed to saying "integrate [product name]"

What about saying something like:
*title*: Add seamless purchasing of crypto assets to your website or app
*description*: Learn how to use 0x Instant or AssetBuyer to allow your users to seamlessly purchase crypto assets using 0x